containers/ubuntu-lts-builder/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apt install -y \
   libqt6svg6-dev libbluetooth-dev libasound2-dev libpulse-dev libgl1-mesa-dev
 
 # Android build dependencies
-RUN apt install -y openjdk-11-jdk-headless
+RUN apt install -y openjdk-11-jdk-headless openjdk-17-jdk-headless
```

Comment:
Do we still need to install OpenJDK 11 as well?

Reply:
We won't once my Gradle PR is merged. But just to fix building that PR now, we could have both and remove 11 later.